release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -467,7 +467,32 @@ In the table, features are marked with the following statuses:
 === Removed features
 
 [id="ocp-4-6-bug-fixes"]
-== Bug fixes
+== Bug fixes 
+
+*Web console (Developer perspective)*
+
+* Previously, when you tried to delete a Knative application through a *Topology* view, a false positive error about an non-existing *Knative route* was reported. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1866214[BZ#1866214])
+
+* Previously, Developer Console didn't allow images from insecure registries to be imported. This bug fix adds a checkbox that allows user to use the insecure registries in the *Deploy image* form (link:https://bugzilla.redhat.com/show_bug.cgi?id=1826740[BZ#1826740])
+
+* When a new application was created through the Developer Console a Developer Catalog was not being displayed. This was caused by an issue in the Jaeger operator. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1845279[BZ#1845279])
+
+* Previously, when modifying a Pipeline through the *Pipeline Builder* in the Developer Console, the interface was rearranging the tasks incorrectly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1856155[BZ#1856155])
+
+* A web console was crashing with a JavaScript exception when the user cancelled the creation of a Pipeline through a web console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1856267[BZ#1856267])
+
+* User having the required permissions was unable to retrieve and deploy an image from an another project. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1843222[BZ#1843222])
+
+* When you tried to deploy an application from a Git repository with *Import from Git* function, even though the repository was reachable by the cluster, the Developer Console reported a false positive error `Git repository is not reachable`. This was fixed by changing the error message with the information on how to make the private repository available to the cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1877739[BZ#1877739])
+
+* When a Go application was created through the Developer Console, a route to the application was not created. This was caused by a bug in `build-tools` and incorrectly configured ports. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1874817[BZ#1874817])
+
+* When you created an application with the *Import from Git* function, a subsequent change of the application's Git repository from the web console was not possible. This was caused by applying the application name in real-time when editing the application's properties. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1873095[BZ#1873095])
+
+* Previously, a user without an administrative or project listing priviledges couldn't see the metrics of any projects. This bug fix removes the checks for user priviledges when accessing the cluster metrics. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1842875[BZ#1842875])
```

Comment:
privileges - change in both the instances
remove an before administrative

Reply:
```suggestion
* Previously, a user without administrative or project listing privileges couldn't see the metrics of any projects. This bug fix removes the checks for user privileges when accessing the cluster metrics. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1842875[BZ#1842875])
```